src/cmd/write.go

```diff
@@ -28,7 +27,7 @@ var writeCmd = &cobra.Command{
 		err = repo.Store(command)
 		tome.Check(err)
 
-		fmt.Printf("Stored command: %s\n", command.String())
+		tome.LogInfof("Stored command: %s\n", command.String())
```

Comment:
The log statement is not relevant for a user. The return code should be enough. So I would rather drop the log rather than sticking to the current behavior. wdyt?

Reply:
fine by me